src/pages/AddGamePage.js

```diff
@@ -1,36 +1,29 @@
 import styled from 'styled-components/macro'
 import Search from '../components/Search'
 
-//const api_key = process.env.REACT_APP_API_KEY;
-
 export default function AddGamePage({ onSubmit, games }) {
   return (
     <Container>
       <Search onSubmit={onSubmit} />
       <List>
         {games.map(game => (
-          <Li backgroundImage={game.background_image}>{game.name}</Li>
+          <ListItem backgroundImage={game.background_image}>
+            {game.name}
+          </ListItem>
         ))}
       </List>
```

Comment:
You could destructure game  to `({background_image, name})`
I prefer to name my styled components like `StyledListItem` or `StyledList` but thats personal preference